public/manifest.json

```diff
@@ -1,19 +1,19 @@
 {
-  "short_name": "React App",
-  "name": "Create React App Sample",
+  "short_name": "Propguard",
+  "name": "Multirotor Robot Design Team Propguard",
   "icons": [
     {
-      "src": "favicon.ico",
+      "src": "multirotor-icon.png",
       "sizes": "64x64 32x32 24x24 16x16",
       "type": "image/x-icon"
     },
     {
-      "src": "logo192.png",
+      "src": "multirotor-icon.png",
       "type": "image/png",
       "sizes": "192x192"
     },
     {
-      "src": "logo512.png",
+      "src": "multirotor-icon.png",
       "type": "image/png",
       "sizes": "512x512"
```

Comment:
With all of these, this is inherently not true, please either create versions at each of these sizes or just leave it as it was